### PR TITLE
Added a .desktop file.

### PR DESCRIPTION
**Pull request by Mailaender**
This should adhere to https://specifications.freedesktop.org/desktop-entry-spec/latest/.

### CODE REVIEW

**Comment by janisozaur** [comment]
That's very nice.

Can you update the packaging part of `CMakeLists.txt` and debian rules as well to include this file?

File: distribution/linux/openrct2.desktop [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Exec=openrct2
+Icon=openrct2
+Name=OpenRCT2
+Comment=Open Roller Coaster Tycoon 2
```
Comment (janisozaur) [inline distribution/linux/openrct2.desktop]: I'd rather this said
> An open source re-implementation of Roller Coaster Tycoon 2.

Just like our project blurb here on GitHub.

Comment (Gymnasiast) [inline distribution/linux/openrct2.desktop]: `An open-source clone of RollerCoaster Tycoon 2` would be better, I think.

**Comment by Gymnasiast** [comment]
Since .desktop files support translation, could you add a translation for German?
Here are the relevant Dutch translations:
```
Comment[nl]=Een opensource-versie van RollerCoaster Tycoon 2.
GenericName[nl]=Pretparksimulator
```

**Comment by janisozaur** [comment]
For reference: https://github.com/OpenTTD/OpenTTD/blob/master/media/openttd.desktop.in

File: distribution/linux/openrct2.desktop (hunk 1 above)
Comment (IntelOrca) [inline distribution/linux/openrct2.desktop]: @Gymnasiast are you saying we have to update all our headers again?

Comment (Gymnasiast) [inline distribution/linux/openrct2.desktop]: @IntelOrca : I fail to see how you would conclude that from what I said. Our own readme calls it a re-implementation, and we're under no obligation to use the same description as we used in the headers.

**Comment by HaasJona** [comment]
German would be

```
Comment[de]=Eine Open-Source-Neuimplementierung von RollerCoaster Tycoon 2
GenericName[de]=Freizeitpark-Simulation
```

**Comment by Manuel-K** [comment]
What about “quelloffene Neuimplementierung” instead of “Open-Source-Neuimplementierung”?

**Comment by Mailaender** [comment]
Updated.

> Can you update the packaging part of CMakeLists.txt and debian rules as well to include this file?

I don't know enough about `cmake` and don't run Debian so I can't update everything. The following shell scripts need to be translated into CMake syntax:

```sh
install -D resources/logo/icon_x16.png $DESTDIR/usr/share/icons/hicolor/16x16/apps/openrct2.png
install -D resources/logo/icon_x32.png $DESTDIR/usr/share/icons/hicolor/32x32/apps/openrct2.png
install -D resources/logo/icon_x64.png $DESTDIR/usr/share/icons/hicolor/64x64/apps/openrct2.png
install -D resources/logo/icon_x128.png $DESTDIR/usr/share/icons/hicolor/128x128/apps/openrct2.png
install -D resources/logo/icon_x256.png $DESTDIR/usr/share/icons/hicolor/256x256/apps/openrct2.png
install -D resources/logo/icon_flag.svg $DESTDIR/usr/share/icons/hicolor/scalable/apps/openrct2.svg

install -D distribution/linux/openrct2.desktop $DESTDIR/usr/share/applications/
```

**Comment by HaasJona** [comment]
@Manuel-K Also possible. Everyone who knows what "quelloffen" means, knows that it means "Open Source", so I don't see much difference. Your's is more "germanized" than mine.

**Comment by Mte90** [comment]
The italian string: `Comment[it]=Versione Open Source di RollerCoaster`

**Comment by Manuel-K** [comment]
@Mailaender Please add back the indefinite article or change the first letter to a capital.

**Comment by Wirlie** [comment]
Spanish:
```
Comment[es]=Re-implementación de código abierto de Roller Coaster Tycoon 2.
GenericName[es]=Simulación de parque de atracciones.
```

**Comment by Mailaender** [comment]
Updated again.

**Comment by LinusU** [comment]
Swedish:

```text
Comment[sv]=Open Source om-implementation av RollerCoaster Tycoon 2.
GenericName[sv]=Nöjesparks-simulator
```

**Comment by Mailaender** [comment]
Updated again. Not sure how many more languages I should add on request. It might be easier if you just merge it and then others append their translations.

**Comment by janisozaur** [comment]
@Gymnasiast how about we merge this and you create an issue over in Localisation repo, so translators would be notified?

**Comment by Gymnasiast** [comment]
@janisozaur That's ok.

**Pull request merged**